### PR TITLE
Fixed formatting in eventPublisher

### PR DESCRIPTION
**Pull request by ashanthamara**
## Purpose
Address the comments 
- https://github.com/wso2/identity-apps/pull/3247#discussion_r1108030028
- https://github.com/wso2/identity-apps/pull/3241#discussion_r1108027266

Related PRs
- https://github.com/wso2/identity-apps/pull/3241
- https://github.com/wso2/identity-apps/pull/3247

### CODE REVIEW

File: apps/console/src/features/applications/components/application-danger-zone.tsx [hunk 1]
```diff
@@ -96,11 +98,12 @@ export const ApplicationDangerZoneComponent: FunctionComponent<ApplicationDanger
...
-                eventPublisher.publish("application-delete", {
-                    "client-id": clientId
-                });
+                eventPublisher.publish(
+                    "application-delete", 
+                    { "client-id": clientId }
+                );
             })
-            .catch((error) => {
+            .catch((error: AxiosError) => {
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/application-danger-zone.tsx:106]: Can we check if this is `IdentityAppsApiException`?

**Review by dasuni-30 — APPROVED** [review]

File: apps/console/src/features/applications/components/application-danger-zone.tsx (hunk 1 above)
Comment (ashanthamara) [reply apps/console/src/features/applications/components/application-danger-zone.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/3829#:~:text=Fixed%20error%20type-,3030a76,-ashanthamara%20dismissed%20dasuni

File: apps/console/src/features/applications/components/application-danger-zone.tsx [hunk 2]
```diff
@@ -52,9 +54,9 @@ interface ApplicationDangerZonePropsInterface extends
 /**
  * Application Danger Zone component
  * 
- * @param { ApplicationDangerZonePropsInterface } props - Props injected to the component.
+ * @param props - Props injected to the component.
  * 
- * @return {React.ReactElement}
+ * @returns ReactElement
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/application-danger-zone.tsx:59]:
```suggestion
 * @returns ApplicationDangerZoneComponent.
```

Comment (ashanthamara) [reply apps/console/src/features/applications/components/application-danger-zone.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/3829/commits/4958e161bc015b284a3358ca83f25c8a3fd6199c

**Review by dasuni-30 — APPROVED** [review]

**Pull request merged**